ferrocene/library/libc/src/lib.rs

```diff
@@ -23,6 +23,7 @@
 )]
 #![cfg_attr(libc_deny_warnings, deny(warnings))]
 // Attributes needed when building as part of the standard library
+#![allow(internal_features)]
```

Comment:
Ah this wasnt a feature hence why my recommendation was wrong, this s hould be
```rs
// ferrocene addition
#![cfg_attr(feature = "rustc-dep-of-std", allow(internal_features))]
```
then (ideally with the comment telling why we need this, I forgot the reason)

Reply:
right, because cfg_link fails in CI because it's an internal feature